verify/functions/start-verify.js

```diff
@@ -31,67 +26,50 @@ exports.handler = function (context, event, callback) {
    * response.appendHeader('Access-Control-Allow-Headers', 'Content-Type');
    */
 
-  if (typeof event.to === 'undefined') {
-    response.setBody({
-      success: false,
-      error: {
-        message: 'Missing parameter; please provide a phone number or email.',
-        moreInfo: 'https://www.twilio.com/docs/verify/api/verification',
-      },
-    });
-    response.setStatusCode(400);
-    return callback(null, response);
-  }
+  try {
+    if (typeof event.to === 'undefined') {
```

Comment:
If a user doesn't enter a phone number or email address then an empty string is sent through and the error happens at the API level. You could change this to `if (typeof event.to === 'undefined' || event.to.trim() === "")` and the error would happen earlier.

Alternatively, you could add a `required` attribute to the phone number or email field when it is visible.